src/pages/AuthorsPage.jsx

```diff
@@ -0,0 +1,8 @@
+const AuthorsPage = () => (
+  <div>
+    <h1>THis is the Authors Page</h1>
+    <button type="button">Check status</button>
+  </div>
+);
+
+export default AuthorsPage;
```

Comment:
In order to respect the logic that this project follows, we need to respect the milestone requirements regarding the names of components and the name of the different sections of our app.

- [ ] Kindly, change the name to this component and file from AuthorsPage to Categories and, kindly, respect the requirements of the project (in future we will have to respect the requirements set by clients) :+1: 

Screenshot from project requirements:
![image](https://user-images.githubusercontent.com/83596135/165865610-5f8a361c-9562-4085-9974-2e4b2c994e8c.png)